cms/models.py

```diff
@@ -214,22 +215,28 @@ class WebinarPage(MetadataPageMixin, Page):
     description = models.TextField(
         null=True, blank=True, help_text="Description of the webinar."
     )
-    action_title = models.CharField(
-        max_length=255,
-        help_text="Specify the webinar call-to-action text here (e.g: 'REGISTER, VIEW RECORDING').",
-    )
     action_url = models.URLField(
         help_text="Specify the webinar action-url here (like a link to an external webinar page).",
+        null=True,
+        blank=True,
+    )
+    sub_heading = models.CharField(
+        max_length=250,
+        null=True,
+        blank=True,
+        help_text="Sub heading of the webinar page.",
     )
+    course = models.ForeignKey(Course, null=True, on_delete=models.DO_NOTHING)
```

Comment:
Course is a required field in CMS, Is it intentional?

Reply:
Based on [discussion](https://github.com/mitodl/hq/issues/1818), I believe every webinar should be linked to some course. That's why it is kept mandatory.

Reply:
I have a question(s) as well:

1. I thought we wanted to associate these with a course or a _**program**_. But I see only course foreign key. why is that? Probably it was discussed somewhere.
2. I'm also not sure why we've kept it mandatory. Where was it discussed? I thought we may or may not associate a webinar with a course/program.
    - I assumed we would not show that course info link at the bottom if there is no course/program associated.
    - Also, `Learn more about this course or enroll` needs to be updated to support for the program, Maybe we can write it as `Learn more about this courseware or enroll`.